concepts/55 Vue Components/05 Add DevExtreme to a Vue Application/40 Import Stylesheets.md

```diff
@@ -5,6 +5,18 @@ Open the main component file (`main.js`) and import `dx.common.css` and a [prede
     import 'devextreme/dist/css/dx.light.css';
     // ...
 
+Then, go to the `public` folder, open the `index.html` file, and add the `dx-viewport` class to the `<body>` tag. This ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets.
```

Comment:
Can we change the second sentence to:
This ensures that theme colors and typography settings are applied to all page elements (and not only to DevExtreme widgets).